flow/FlowStreamCreator.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { UD21x18 } from "@prb/math/src/UD21x18.sol";
+
+import { ISablierFlow } from "@sablier/flow/src/interfaces/ISablierFlow.sol";
+
+import { FlowUtilities } from "./FlowUtilities.sol";
+
+contract FlowStreamCreator {
+    // Mainnet USDC
+    IERC20 public constant USDC = IERC20(0xf08A50178dfcDe18524640EA6618a1f965821715);
+    ISablierFlow public immutable sablierFlow;
+
+    constructor(ISablierFlow sablierFlow_) {
+        sablierFlow = sablierFlow_;
+    }
+
+    // Create a stream that sends 1000 USDC per month
+    function createStream_1T_PerMonth() external returns (uint256 streamId) {
```

Comment:
should be `createStream_1K_PerMonth`. T is best suited to represent Trillions.

Reply:
https://github.com/sablier-labs/examples/pull/37/commits/9fedb7ad0f2ce57842e8b6e3da3999c39b935e7d